packages/strapi-design-system/createIndexFiles.js

```diff
@@ -1,8 +1,10 @@
 const fs = require('fs-extra');
 const path = require('path');
 
-const excludedFolders = ['helpers', '.DS_Store'];
-const fileNames = fs.readdirSync(path.resolve(__dirname, 'src'));
+const excludedFolders = process.env.IS_V2 === 'false' ? ['helpers', '.DS_Store'] : ['helpers', '.DS_Store', 'v2'];
```

Comment:
Can we create a constant `isV2Build` ? 

```
const isV2Build = process.env.IS_V2 === 'true'
const excludedFolders = ['helpers', '.DS_Store'];
let entryFolder = path.resolve(__dirname, 'src');
let outputPath = path.resolve(__dirname, 'dist', `${name}.js`);

if (isV2Build) {
 excludedFolders.push('v2')
 entryFolder = path.resolve(__dirname, 'src', 'v2');
outputPath = ...
}
```

Also why are you using the `=== 'false'` instead of the `'true'` one here?